script/upgrades/MU01/governance/MU01-CGP-Phase2.sol

```diff
@@ -238,6 +256,141 @@ contract MU01_CGP_Phase2 is ICGPBuilder, GovernanceScript {
     }
   }
 
+  function proposal_configureBreakerBox() public {
+    // Add the Median Delta Breaker to the breaker box with the trading mode '1' -> No Trading
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(medianDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, medianDeltaBreaker, 1)
+        )
+      );
+    }
+
+    // Add the Value Delta Breaker to the breaker box with the trading mode '2' -> No Trading
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(valueDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, valueDeltaBreaker, 2)
+        )
+      );
+    }
+
+    // Set rate feed dependencies cEUR/USDC, cBRL/USDC for cUSD/USDC rate feed
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        breakerBox,
+        abi.encodeWithSelector(
+          BreakerBox(0).setRateFeedDependencies.selector,
```

Comment:
I think this is not doing the correct thing:
 we want:
 ` setRateFeedDependencies(cEURUSDCConfig.referenceRateFeedID, [cUSDUSDCConfig.referenceRateFeedID])`
and 
` setRateFeedDependencies(cBRLUSDCConfig.referenceRateFeedID, [cUSDUSDCConfig.referenceRateFeedID])`
based on the breaker box function:
`function setRateFeedDependencies(address rateFeedID, address[] memory dependencies)`
right?